src/screens/Project/projectPage.module.scss

```diff
@@ -0,0 +1,55 @@
+@import "../../styles/Colors.scss";
```

Comment:
Rename this file to `ProjectPage.module.scss` (the first letter needs to be uppercase), its failing deployment on the server